### PR TITLE
Fix best practices example in the `basics\pipe-operator` lesson

### PR DESCRIPTION
**Pull request by eze1981**

### CODE REVIEW

File: lessons/basics/pipe-operator.md [hunk 1]
```diff
@@ -53,18 +53,17 @@ true
 
 ## Best Practices
 
-If the arity of a function is more than 1, then make sure to use parenthesis. This doesn't matter much to the Elixir, but it matters to other programmers who may misinterpret your code. If we take our 2nd example, and remove the brackets from `Enum.map/2`, we are met with the following warning.
+If the arity of a function is more than 1, then make sure to use parenthesis. This doesn't matter much to the Elixir, but it matters to other programmers who may misinterpret your code. If we take our 3rd example, and remove the brackets from `String.ends_with?/2`, we are met with the following warning.
```
Comment (ybur-yug) [inline lessons/basics/pipe-operator.md]: `parenthesis` should be `parentheses`. I wasnt sure but checked - [source](http://www.dictionary.com/browse/parentheses)

**Comment by ybur-yug** [comment]
Looks good to me if the minor correction RE `parenthesis` is corrected @eze1981

**Comment by doomspork** [comment]
@eze1981 I think this is a great improvement and something we should make sure is applied to all of the translations.  Could you take a peek at doing just that?

**Comment by devleoper** [comment]
I think we should make an issue for applying this change to the translations.

We can easily change `2nd` into `3rd` for [some translations](https://github.com/doomspork/elixir-school/blob/master/jp/lessons/basics/pipe-operator.md#%E3%83%99%E3%82%B9%E3%83%88%E3%83%97%E3%83%A9%E3%82%AF%E3%83%86%E3%82%A3%E3%82%B9), but [other](https://github.com/doomspork/elixir-school/blob/master/ko/lessons/basics/pipe-operator.md#%EC%A2%8B%EC%9D%80-%EC%8A%B5%EA%B4%80) [translations](https://github.com/doomspork/elixir-school/blob/master/es/lessons/basics/pipe-operator.md#buenas-pr%C3%A1cticas) which translated ordinal numbers to their expressions needs some help from translators.

**Comment by ybur-yug** [comment]
Should we wait to merge this until the translations are all good, or since the issue is created just merge it now?

**Comment by doomspork** [comment]
This is good to merge @ybur-yug, we'll make sure it gets translated now.

Thanks again @eze1981 👍

**Pull request merged**